release_management/python-releases.toml

```
@@ -0,0 +1,3501 @@
+# This file is placed in the public domain or under the
+# CC0-1.0-Universal licence, whichever is more permissive.
+#
+# This document contains the history of every release of the Python project,
+# and specifically the CPython intepreter. The data in this file were initially
+# compiled in 2025 by Adam Turner, with information primarily sourced from the
+# release PEPs and supplemented by the 'releases' section of www.python.org.
+#
+# The release schedules for Python 3.8 onwards are created from data in this
+# document. After editing this file, run the following command to regenerate
+# the relevant PEPs:
+#
+#    python -m release_management update-peps
+#
+# The PEP rendering system, via Sphinx, uses this document to regenerate the
+# 'release-cycle' JSON file, found at https://peps.python.org/release-cycle.json,
+# and a full JSON representation at https://peps.python.org/python-releases.json,
+# This 'release-cycle' JSON file is intended for public consumption. The format
+# of this TOML document is not guaranteed and may change without notice.
+
+# -- Python 1.6 --------------------------------------------------------------
+
+# Drake wrote PEP 160, but Hylton served as release manager:
+# https://mail.python.org/archives/list/python-dev@python.org/message/FCPLKMFDZUDOQGPAEKGKB2VQYTI4JT7Y/
+# https://jeremyhylton.blogspot.com/2006/05/contributing-to-python.html
+
+[metadata."1.6"]
+pep = 160
+status = "end-of-life"
+branch = ""  # no branch or tag for 1.6 exists
+release-manager = "Guido van Rossum & Jeremy Hylton"
```

CSV, like we have for 2.3 (3 RMs) and 3.3 (2 RMs)?
```suggestion
release-manager = "Guido van Rossum, Jeremy Hylton"
```

---

This is a format for machines to consume, would it be easier to deal with CSV than write a custom parser needing to handle "and" and ampersands?

(But I don't see a pressing need for splitting this list, so not a blocker.)